src/index.tsx

```diff
@@ -6,8 +6,8 @@ import "./index.css";
 // This enables HMR
 // TODO: check if it affects prod build
 // sorry TS
-if (((module as unknown) as Record<string, unknown>)?.hot) {
-  ((module as unknown) as Record<"hot", { accept: () => void }>)?.hot?.accept();
+if ((module as unknown as Record<string, unknown>)?.hot) {
```

Comment:
TS was crying during compilation :P If these unknowns necessary - ok, but some comment here would be nice.